src/Utils.cpp

```diff
@@ -122,6 +122,17 @@ void Utils::gtp_printf(int id, const char *fmt, ...) {
     va_end(ap);
 }
 
+void Utils::gtp_printf_raw(const char *fmt, ...) {
```

Comment:
Probably many of the functions in this file should be rewritten to call this instead of duplicating it.

Reply:
Agreed